src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs

```diff
@@ -36,6 +36,7 @@ internal abstract class RegexCompiler
         private static readonly MethodInfo s_charInClassMethod = RegexRunnerMethod("CharInClass");
         private static readonly MethodInfo s_checkTimeoutMethod = RegexRunnerMethod("CheckTimeout");
 
+        private static readonly MethodInfo s_regexCaseEquivalencesTryFindCaseEquivalencesForCharWithIBehaviorMethod = typeof(RegexCaseEquivalences).GetMethod("TryFindCaseEquivalencesForCharWithIBehavior", BindingFlags.Static | BindingFlags.Public)!;
```

Comment:
>s_regexCaseEquivalencesTryFindCaseEquivalencesForCharWithIBehaviorMethod [](http://example.com/codeflow?start=43&length=72)

Nice! Maybe this is a record for the longest name in the framework?  :smile:  Just kidding.

Reply:
lol, I was thinking about it while I was typing it. I could simplify the name if desired of course.

Reply:
https://github.com/dotnet/runtime/blob/67761043f1b0e1514a1b8c1c6d99a353737a24a1/src/libraries/System.Composition/tests/ConstraintTests.cs#L44

Reply:
oh bummer! I'll have to work harder then in order to get the record...